osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/WatchHighScoresCommand.cs

```diff
@@ -132,6 +132,15 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                                              .OrderBy(s => s.queue_id)
                                              .ToArray();
 
+                    var pendingProcessing = highScores.FirstOrDefault(s => s.status == 0);
```

Comment:
I was side-eyeing this because `status` is declared as `byte?` so wasn't sure whether it was possible to have it `null`. But it's not, because the query is a `SELECT * FROM osu.score_process_queue`, so if a row is returned here, it _must_ have the `score_process_queue` members populated